membership-attribute-service/app/controllers/AttributeController.scala

```diff
@@ -274,4 +274,10 @@ class AttributeController(
     else
       maybeAttributes
   }
+
+  def isTestUser: Action[AnyContent] =
+    AuthorizeForScopes(List(readSelf)) {
+      NoContent
+    }
```

Comment:
this will return something like this (due to the AddGuIdentityHeadersFilter that applies to all endpoints)
x-gu-identity-id: NNNNNN
x-gu-membership-test-user: false